Terminal.Gui/Application.cs

```diff
@@ -622,7 +599,19 @@ public static void Run<T> (Func<Exception, bool> errorHandler = null, ConsoleDri
     ///     RELEASE builds only: Handler for any unhandled exceptions (resumes when returns true,
     ///     rethrows when null).
     /// </param>
-    public static void Run (Toplevel view, Func<Exception, bool> errorHandler = null)
+    /// <param name="driver">
+    ///     The <see cref="ConsoleDriver"/> to use. If not specified the default driver for the platform will
+    ///     be used ( <see cref="WindowsDriver"/>, <see cref="CursesDriver"/>, or <see cref="NetDriver"/>). Must be
+    ///     <see langword="null"/> if <see cref="Init"/> has already been called.
+    /// </param>
+    public static void Run (Toplevel view, Func<Exception, bool> errorHandler = null, ConsoleDriver driver = null)
+    {
+        EnsureValidInitialization (view, driver);
+
+        RunApp (view, errorHandler);
+    }
+
+    private static void RunApp (Toplevel view, Func<Exception, bool> errorHandler = null)
```

Comment:
Because I want to ensure that the toplevel has a valid initialization. Before it was checking twice. If a user call `Run(v)` I want to ensure that the initialization is valid. If a user call `Run<T>()` the validation is called and when is passed to `Run(v)` it check the validation again. I tried to not do that but I finished thinking it's the better way. Unless something is escape me.